tests/conftest.py

```diff
@@ -112,6 +113,8 @@ def fake_build_plan() -> list[models.BuildInfo]:
 @pytest.fixture()
 def enable_overlay() -> Iterator[craft_parts.Features]:
     """Enable the overlay feature in craft_parts for the relevant test."""
+    if not shutil.which("fuse-overlayfs"):
```

Comment:
```suggestion
    if os.getenv("CI") != "true" and not shutil.which("fuse-overlayfs"):
```
... or equivalent, so that we don't disable these tests on CI by mistake in the future

Reply:
Good point, thanks!